tests/orion/api/test_block_types.py

```diff
@@ -394,11 +405,19 @@ async def test_delete_nonexistent_block_type(self, client):
         response = await client.delete(f"/block_types/{uuid4()}")
         assert response.status_code == status.HTTP_404_NOT_FOUND
 
-    async def test_delete_system_block_type_fails(self, client, system_block_type):
+    async def test_delete_system_block_type_fails(self, system_block_type, client):
         response = await client.delete(f"/block_types/{system_block_type.id}")
         assert response.status_code == status.HTTP_403_FORBIDDEN
         assert response.json()["detail"] == "protected block types cannot be deleted."
 
+    async def test_delete_system_block_type_does_not_fail_for_older_clients(
+        self, system_block_type, client_with_unprotected_block_api
+    ):
+        response = await client_with_unprotected_block_api.delete(
+            f"/block_types/{system_block_type.id}"
+        )
+        assert response.status_code != status.HTTP_403_FORBIDDEN
```

Comment:
Should we assert that status code is 204 like the other tests that have a successful deletion?